components/wifi/esp8266-driver/ESP8266Interface.h

```diff
@@ -78,7 +78,7 @@ class ESP8266Interface : public NetworkStack, public WiFiInterface {
      * @param rx        RX pin
      * @param debug     Enable debugging
      */
-    ESP8266Interface(PinName tx, PinName rx, bool debug = false, PinName rts = NC, PinName cts = NC, PinName rst = NC);
+    ESP8266Interface(PinName tx, PinName rx, bool debug = false, PinName rts = NC, PinName cts = NC, PinName rst = NC, PinName pwr = NC);
```

Comment:
This is a binary compatibility break, but I guess this isn't a core enough component for that to be an issue.